packages/admin-next/dashboard/src/routes/orders/order-detail/order-detail.tsx

```diff
@@ -0,0 +1,52 @@
+import { useAdminOrder } from "medusa-react"
+import { useLoaderData, useParams } from "react-router-dom"
+import { JsonViewSection } from "../../../components/common/json-view-section"
+import { OrderCustomerSection } from "./components/order-customer-section"
+import { OrderFulfillmentSection } from "./components/order-fulfillment-section"
+import { OrderGeneralSection } from "./components/order-general-section"
+import { OrderPaymentSection } from "./components/order-payment-section"
+import { OrderSummarySection } from "./components/order-summary-section"
+import { orderExpand } from "./constants"
+import { orderLoader } from "./loader"
+
+export const OrderDetail = () => {
+  const initialData = useLoaderData() as Awaited<ReturnType<typeof orderLoader>>
+
+  const { id } = useParams()
+
+  const { order, isLoading, isError, error } = useAdminOrder(
+    id!,
+    {
+      expand: orderExpand,
+    },
+    {
+      initialData,
+    }
+  )
+
+  if (isLoading || !order) {
+    return <div>Loading...</div>
+  }
+
+  if (isError) {
+    throw error
+  }
+
+  return (
+    <div className="grid grid-cols-1 gap-x-4 xl:grid-cols-[1fr,400px]">
+      <div className="flex flex-col gap-y-2">
```

Comment:
q: do we miss widget slots here?

Reply:
Yea, waiting with it for the later, as I think we are properly going to add some more injection zones then what we have today